ReactWindows/ReactNative.Net46.Tests/Internal/DispatcherHelpers.cs

```diff
@@ -14,7 +14,7 @@ static class DispatcherHelpers
             Dispatcher dispatcher = Application.Current != null
                 ? Application.Current.Dispatcher
                 : Dispatcher.CurrentDispatcher;
```

Comment:
The UWP tests use an application for some of its tests, so if we follow the same pattern with Net46 then the Application.Current might be relevant... However, if we are never going to run an Application to test then the Dispatcher.Current is all that is needed...